src/FilamentDeveloperLoginsPlugin.php

```diff
@@ -31,10 +35,10 @@ class FilamentDeveloperLoginsPlugin implements Plugin
 
     public string $column = 'email';
 
-    /**
-     * @var class-string<\Illuminate\Database\Eloquent\Model&\Illuminate\Contracts\Auth\Authenticatable>
-     */
-    public string $modelClass = User::class;
+    public function __construct()
+    {
+        $this->modelClass = config('auth.providers.users.model') ?? \App\Models\User::class;
```

Comment:
Please add a use statement.

```suggestion
        $this->modelClass = config('auth.providers.users.model') ?? User::class;
```